docs/app/pages/components/components-sections/notifications/notifications/snippets/app.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app',
+    templateUrl: './app.component.html'
+})
+export class AppComponent {
+
+}
+
+interface NotificationRef {
```

Comment:
These should not be defined here, they should be imported from '@ux-aspects/ux-aspects'